main.cc

```diff
@@ -402,6 +402,10 @@ int main( int argc, char ** argv )
   app.setStyle(new GdAppStyle);
 #endif
 
+#if defined( Q_OS_LINUX ) && QT_VERSION >= QT_VERSION_CHECK( 5, 7, 0 )
```

Comment:
Don't limit to `Q_OS_LINUX`. While calling this function is not useful on Windows and macOS, it still compiles and won't harm. And it _should_ be called on BSD systems.